hw/ip/otbn/data/insns.yml

```
@@ -491,6 +473,72 @@ insns:
       elif writeback_variant == 'writeout':
         WDR[d] = ACC
 
+  - mnemonic: bn.mulqacc.wo
+    group: bignum
+    synopsis: Quarter-word Multiply and Accumulate with half-word writeback
+    operands:
+      - *mulqacc-zero-acc
+      - &mulqacc-wrd
+        name: wrd
+        doc: Destination WDR.
+      - *mulqacc-wrs1
+      - *mulqacc-wrs1-qwsel
+      - *mulqacc-wrs2
+      - *mulqacc-wrs2-qwsel
+      - *mulqacc-acc-shift-imm
+    syntax: |
+      [<zero_acc>] <wrd>, <wrs1>.<wrs1_qwsel>, <wrs2>.<wrs2_qwsel>, <acc_shift_imm>
+    doc: |
+      Multiplies two `WLEN/4` WDR values and adds the result to an accumulator after shifting it.
+      Writes the resulting accumulator to <wrd>.
```

```suggestion
      Writes the resulting accumulator to `<wrd>`.
```

(Otherwise it's a HTML tag and not rendered.)

Not sure what the solution would be (or if we can find an easy one), but it would be potentially nice to get an indication that `doc` requires a different escaping/syntax than the `syntax` section above it.

---

Oops, good point. Fixed. For the general "documentation" of the YAML format, there are some comments about the file format at the top of the file. I'm not sure we need much more: after all, if someone makes a small change to a documentation string, they'll presumably regenerate the HTML and eyeball it to check it did what they expected.